src/Components/QuizContainer/QuizInstructions.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@chakra-ui/button';
+import { StarIcon } from '@chakra-ui/icons';
+
+import {
+	Container,
+	Heading,
+	List,
+	ListIcon,
+	ListItem,
+} from '@chakra-ui/layout';
+import { btnStyleProps, solidPrimaryButtonStyleProps } from '../utils';
+import { QuizInstructionsProps } from './QuizInstructionsProps.type';
+import { instructionsContainerStyleProps, listStyleProps } from './utils';
+
+export const QuizInstructions = ({
+	quiz,
```

Comment:
here you are consuming only "name" from the quiz so pass only the quiz name

Reply:
Yeah that makes sense, updated.